Indexer/Indexer.php

```diff
@@ -70,8 +70,14 @@ class Indexer
 
     private $em;
 
-    public function __construct()
+    /**
+     * @var LoaderInterface
+     */
+    private $loaderChain;
```

Comment:
@alcaeus it IS a chain loader, you could have xml/yml/annotation configuration in the same project on different bundles